src/nvim/ex_getln.c

```diff
@@ -1477,6 +1480,11 @@ static int command_line_handle_key(CommandLineState *s)
       return command_line_not_changed(s);                   // Ignore mouse
     }
 
+    if (s->firstc == '#' && mouse_row < cmdline_row) {
```

Comment:
We probably only want to do this for press events, not DRAG or RELEASE (it should then fallthrough to the ordinary code below)

Reply:
Something like this?
```c
if (s->firstc == '#' && mouse_row < cmdline_row && s->ignore_drag_release)
```
It does solve the problem of returning as soon as mouse is pressed. It returns after mouse is released.

Reply:
I don't think that is a problem, activation on press is expected. The situation is rather to press mouse at cmdline (which moves the cursor), and then accidentially drag over the second last line, which should continue to move the cursor, not accept the last choice.

Reply:
After the change mentioned above, if you drag over the second last line from the cmdline, it won't be accepted but if you release while on it, it will be. Is this the expected behavior?

Reply:
hmm I will try it

Reply:
To me it feels a bit weird, better to introduce `bool is_press = s->c == K_LEFTMOUSE || s->c == K_RIGHTMOUSE` and use it.